### DISCUSSION TITLE
How do I query account details？

### DISCUSSION THREAD

**Discussion by PerrywLiu** [filecoin-project/lotus#7843] [General Q&A]
I can query it with browser, but it always missing some imporant message, may be can I query these mesaage with API? How can I do it ?

**Comment by LexLuthr** [comment]
@PerrywLiu What account details are we talking about here? Can you give an screenshot of missing details from the browser?
It would be ideal if you can also point out what details are you hoping to query so I can look up the API for it.

**Reply by LexLuthr** [reply]
I am archiving this discussion as there has been no new updates from the author.
Please refer to https://github.com/filecoin-project/lotus/discussions/7228 for the discussion guidelines.
